mlflow/tracing/fluent.py

```diff
@@ -549,3 +550,133 @@ def f():
         return TRACE_BUFFER.get(last_active_request_id)
     else:
         return None
+
+
+@experimental
+def use_remote_trace(trace: Trace):
```

Comment:
Do we need to make the public interface require a user to define an instance of `Trace` ? Can we just accept a `Dict` as input and build the `Trace` object for them?

Reply:
(or make it a Union at least)

Reply:
`Union` makes 1100% sense!